### PR TITLE
Update Keycloak to 7.0.0

### PR DESCRIPTION
**Pull request by DanielFran**
Fix https://github.com/jhipster/generator-jhipster/issues/10288

-   Please make sure the below checklist is followed for Pull Requests.

-   [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/docker-compose/templates/realm-config/jhipster-realm.json.ejs [hunk 1]
```diff
@@ -1979,6 +1979,6 @@
     "waitIncrementSeconds": "60",
     "offlineSessionMaxLifespanEnabled": "false"
   },
-  "keycloakVersion": "6.0.1",
+  "keycloakVersion": "7.0.0",
```
Comment (vishal423) [inline generators/docker-compose/templates/realm-config/jhipster-realm.json.ejs]: are there no structural changes in this upgrade?

Comment (vishal423) [reply generators/docker-compose/templates/realm-config/jhipster-realm.json.ejs]: You can import the existing json and then export to check difference in configurations

**Comment by atomfrede** [comment]
Is there anything left to do here (except resolving the conflicts?)

**Comment by DanielFran** [comment]
@atomfrede It might be needed to update realm-config but I do not have the aknowlewdge to implement this...

**Comment by vishal423** [comment]
@DanielFran, let me know if you want me to take it up

**Comment by DanielFran** [comment]
@vishal423 Yes please.

**Comment by vishal423** [comment]
@DanielFran, I have raised PR #10553 and I think we can close this one

**Comment by DanielFran** [comment]
@vishal423 thanks

**Pull request closed without merging**